fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/view/LaoView.java

```diff
@@ -0,0 +1,91 @@
+package com.github.dedis.popstellar.model.objects.view;
+
+import com.github.dedis.popstellar.model.objects.*;
+import com.github.dedis.popstellar.model.objects.security.MessageID;
+import com.github.dedis.popstellar.model.objects.security.PublicKey;
+
+import java.util.*;
+
+public final class LaoView {
+
+  private final Lao lao;
+
+  /**
+   * This class offers useful getters for Lao state to handlers and prevents changing its state. It
+   * is provided as an intermediate step towards functional handling of Objects. To change the state
+   * of an Lao, one can use createLaoCopy() which returns a copy of the wrapped Lao, and update the
+   * repository with said updated LAO.
+   *
+   * @param lao the Lao to be wrapped
+   */
+  public LaoView(Lao lao) {
+    if (lao == null) {
+      throw new IllegalArgumentException();
+    }
+    this.lao = new Lao(lao);
+  }
+
+  public Lao createLaoCopy() {
+    return new Lao(lao);
```

Comment:
If we already create a copy at construction, is it really necessary to create another one here ?

(I would suggest creating the copy only in the `createLaoCopy()` function, not at creation)

Reply:
I think it should stay that way so that later change to the instance of Lao passed to the constructor does not affect the attribute of LaoView.
Same with `createLaoCopy`